assets/utils.tsx

```diff
@@ -759,6 +759,18 @@ export function parseISODate(date?: string): number {
 }
 
 // Will return number of seconds since epoch for today at 00:00:00
-export function getTodayTimestamp(): number {
+function getTodayTimestamp(): number {
```

Comment:
I'm thinking why are we working with UTC rather than local time? Even if date on server is stored in UTC, if we do `new Date(utcString)` or something similar and continue in local time.

Reply:
It depends if we want UI in server time or user time. I'd say we want user time in most cases. Do we pause notifications according to server time or user timezone?

Reply:
we do it on the server based on the user timezone, but not the browser timezone rather the one selected in the dropdown